packages/shared/src/telemetry/attributes.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * The attributes of an OpenTelemetry compliant event for flag evaluation.
+ * @see https://opentelemetry.io/docs/specs/semconv/feature-flags/feature-flags-logs/
+ */
+export const TELEMETRY_ATTRIBUTE = {
+  /**
+   * The lookup key of the feature flag.
+   *
+   * - type: `string`
+   * - requirement level: `required`
+   * - example: `logo-color`
+   */
+  KEY: 'feature_flag.key',
+  /**
+   * Describes a class of error the operation ended with.
+   *
+   * - type: `string`
+   * - requirement level: `conditionally required`
+   * - example: `flag_not_found`
+   */
+  ERROR_CODE: 'error.type',
```

Comment:
one is code the other is type. is that intentional?

Reply:
Yes it was intentional but I changed it anyways. The difference will now be in the hook instead of the attribute const.